libraries/AP_EFI/AP_EFI_Serial_Lutan.h

```diff
@@ -0,0 +1,83 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  support for Lutan serial EFI
+ */
+#pragma once
+
+#include "AP_EFI.h"
+#include "AP_EFI_Backend.h"
+
+class AP_EFI_Serial_Lutan: public AP_EFI_Backend {
```

Comment:
This definition should be gated on `AP_EFI_ENABLED` (or preferable `AP_EFI_SERIAL_LUTAN_ENABLED`...)